cmd/veneur-emit/main.go

```diff
@@ -170,30 +230,131 @@ func sendSpan(conn MinimalConn, span *ssf.SSFSpan) error {
 	return nil
 }
 
-func sendMetrics(client MinimalClient, passedFlags map[string]bool, name string, tags []string) error {
+// TODO: validity checking? or just fire and forget
```

Comment:
For example, priority can either be `normal` or `low`, and I'm not sure if we should check that on the emit side or just build the packet and then send it.